website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -224,18 +229,18 @@ This PATCH endpoint requires a JSON object with the following properties as a re
 
 Properties without a default value are required.
 
-Key path | Type | Default | Description
---- | --- | --- | ---
-`data.type` | string | | Must be `"no-code-modules"`.
-`data.attributes.version-pin` | string   | (previous value) | The module version to use in no-code provisioning workflows.
-`data.attributes.enabled` | boolean   | (previous value) | Set to `true` to enable no-code provisioning workflows, or `false` to disable them.
-`data.relationships.registry-module.data.id` | string | (previous value) | The ID of a module in the organization's Private Registry.
-`data.relationships.registry-module.data.type` | string | (previous value) | Must be `"registry-module"`.
-`data.relationships.variable-options.data[].id` | string | (new value) | The ID of an existing variable-options set. If provided, a new variable-options set replaces the set with this ID. If not provided, this creates a new variable-option set.
-`data.relationships.variable-options.data[].type` | string | | Must be `"variable-options"`.
-`data.relationships.variable-options.data[].attributes.variable-name` | string | | The name of a variable within the module.
-`data.relationships.variable-options.data[].attributes.variable-type` | string | | The data type for the variable. Can be [any type supported by Terraform](/terraform/language/expressions/types#types).
-`data.relationships.variable-options.data[].attributes.options` | Any[] | | A list of allowed values for the variable.
+| Key path | Type | Default | Description |
+| --- | --- | --- | --- |
+| `data.type` | string | | Must be `"no-code-modules"`. |
+| `data.attributes.version-pin` | string   | (previous value) | The module version to use in no-code provisioning workflows. |
```

Comment:
```suggestion
| `data.attributes.version-pin` | string   | Existing value | The module version to use in no-code provisioning workflows. |
```
Correct? We mean that TF uses the value that is already there if you don't supply this attribute, yeah? Also, I don't see a reason put it in parentheses.